### PR TITLE
extra options for the telescope integration

### PR DESCRIPTION
**Pull request by emmanueltouzery**
Hi! Great plugin!

I'd be interested in some extra customization for the telescope integration... In general telescope allows most things to be customizable through the opts parameter.

I've tried to make the entire make_entry customizable, but then it pulled half of the integration in my config file, because it pulls in make_display, displayer and so on.

So for now I came up with this simple approach, which is maybe not powerful enough (with this granular way, there may be a need for further integration points).

A way to make it more extensible without multiple extension points would be to use less local functions, but I understand you do that to achieve lazy loading, so I don't have any better idea..

As for the local modifications I'd like to achieve with these extension points: width 2 instead of 4 for the first column in the displayer.. and for the entry text have nesting.. but only a single level (not recursive), and only if the parent name is shorter than 20 characters.

### CODE REVIEW

File: lua/telescope/_extensions/aerial.lua [hunk 1]
```diff
@@ -59,12 +60,16 @@ local function aerial_picker(opts)
...
-    local name = item.name
-    if show_nesting then
-      local cur = item.parent
-      while cur do
-        name = string.format("%s.%s", cur.name, name)
-        cur = cur.parent
+    if opts.get_entry_text ~= nil then
+      name = opts.get_entry_text(item)
+    else
+      local name = item.name
```
Comment (stevearc) [inline lua/telescope/_extensions/aerial.lua:66]: You need to add a `local name` before the `if...else` and remove the `local` here

**Review by stevearc — CHANGES_REQUESTED** [review]
Approach looks fine! There's one bug and then it's good to merge

File: lua/telescope/_extensions/aerial.lua (hunk 1 above)
Comment (emmanueltouzery) [reply lua/telescope/_extensions/aerial.lua]: done now... initialized it to item.name.. in theory I could initialize to nil or not initialize at all.. not sure what's best.

Comment (stevearc) [reply lua/telescope/_extensions/aerial.lua]: There's basically no difference.

**Review by stevearc — APPROVED** [review]
Thanks for the PR!

**Pull request merged**